charts/fluentd-elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 4.6.0
+version: 4.6.1
```

Comment:
please raise minor version as this a feature not a bugfix.

Reply:
got it. I'll just close this PR and open another one to fix the DCO issue.